pkg/config/environment_detection.go

```diff
@@ -107,6 +107,7 @@ func detectFeatures() {
 	newFeatures := make(FeatureMap)
 	if IsAutoconfigEnabled() {
 		detectContainerFeatures(newFeatures)
+		detectRemoteProcessCollector(newFeatures)
```

Comment:
Either it's a `container` feature and it's in `detectContainerFeatures` and `environment_containers.go` file.
Either it's not and then you can a dedicated file along side the dedicated function call you have here.